xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-test/xwiki-platform-appwithinminutes-test-pageobjects/src/main/java/org/xwiki/appwithinminutes/test/po/ApplicationNameConstants.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.appwithinminutes.test.po;
+
+/**
+ * Stores the constant strings related to pages involving application names.
+ *
+ * @version $Id$
+ * @since 13.2
+ * @since 12.10.6
+ */
+public interface ApplicationNameConstants
+{
+    /**
+     * The error message displayed when we try to create an application with an empty name.
+     */
+    String EMPTY_APP_NAME_ERROR_MESSAGE = "Please enter the application name.";
+    /**
+     * The warning message displayed when we input the name of an existing application.
+     */
+    String APP_NAME_USED_WARNING_MESSAGE = "This application already exists.";
+}
```

Comment:
Not sure if it is the best way to do it but we need to store this constant in this module since it is still used in `xwiki-platform-distribution-flavor-test-ui` and would be duplicated otherwise. WDYT?

Reply:
The constants are ok but why not putting them directly in ApplicationCreatePage PO? Not a big fan of having an interface just for that.

Reply:
BTW note that storing constant in interface is an anti-pattern according to SonarQube, see https://rules.sonarsource.com/java/tag/bad-practice/RSPEC-1214

Reply:
Good to know. I'll move those to the most relevant POs (`ApplicationCreatePage` and `ApplicationEditPage`)

Reply:
Examples:
* https://sonarcloud.io/project/issues?id=org.xwiki.commons%3Axwiki-commons&resolved=false&rules=java%3AS1214
* https://sonarcloud.io/project/issues?id=org.xwiki.rendering%3Axwiki-rendering&resolved=false&rules=java%3AS1214
* https://sonarcloud.io/project/issues?id=org.xwiki.platform%3Axwiki-platform&resolved=false&rules=java%3AS1214

62 issues in platform ;)